sql/core/src/test/scala/org/apache/spark/sql/connector/GroupBasedDeleteFromTableSuite.scala

```diff
@@ -96,27 +96,14 @@ class GroupBasedDeleteFromTableSuite extends DeleteFromTableSuiteBase {
     }
   }
 
-  test("delete runtime group filtering (DPP enabled)") {
-    withSQLConf(SQLConf.DYNAMIC_PARTITION_PRUNING_ENABLED.key -> "true") {
-      checkDeleteRuntimeGroupFiltering()
-    }
-  }
-
-  test("delete runtime group filtering (DPP disabled)") {
-    withSQLConf(SQLConf.DYNAMIC_PARTITION_PRUNING_ENABLED.key -> "false") {
-      checkDeleteRuntimeGroupFiltering()
-    }
-  }
-
-  test("delete runtime group filtering (AQE enabled)") {
-    withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true") {
-      checkDeleteRuntimeGroupFiltering()
-    }
-  }
-
-  test("delete runtime group filtering (AQE disabled)") {
-    withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "false") {
-      checkDeleteRuntimeGroupFiltering()
+  test("delete runtime group filtering") {
```

Comment:
When the test fails, we won't know if AQE or DPP is enabled or disabled?

Reply:
Could we add logs?